Sources/inc/CosmicCORSIKA.hh

```diff
@@ -229,30 +234,35 @@ namespace mu2e {
       static constexpr float _GeV2MeV = CLHEP::GeV / CLHEP::MeV;
       static constexpr float _cm2mm = CLHEP::cm / CLHEP::mm;
       static constexpr float _ns2s = CLHEP::ns / CLHEP::s;
+      static constexpr unsigned _fbsize_words = 5733 + 2;
 
       CLHEP::Hep3Vector _cosmicReferencePointInMu2e;
       float _fluxConstant = 1.8e4;
```

Comment:
I  wanted to know why 1.8e4, but then realized that the number is always owerwritten by the corresponding setting in the fcl file.   May be just get rid of the number here?

Reply:
That comment went onto a wrong line.  It was about the _fluxConstant.

Reply:
1.8e4 is the intensity of cosmic rays primary nucleons as quoted by the PDG and it's needed for the live-time calculation. I would still keep it in the code for clarity. I added a comment.